test/test_blocks.py

```diff
@@ -465,6 +469,57 @@ def test_raise_error_if_event_queued_but_queue_not_enabled(self):
         demo.close()
 
 
+class TestTempFile:
+    def test_pil_images_hashed(self):
+        def create_images(n_images):
+
+            a = Image.new("RGB", (512, 512), "red")
+            b = Image.new("RGB", (512, 512), "green")
+            c = Image.new("RGB", (512, 512), "blue")
```

Comment:
You might want to move these outside `create_images` to save some cycles.

Reply:
I think it's ok for now in this simple test